.github/workflows/ci.yml

```diff
@@ -9,19 +9,15 @@ on:
 jobs:
   build:
     name: CI
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
If there's no incompatibility we should do `ubuntu-22.04` since that's newer

Reply:
Unfortunately, ubuntu 22.04 only [supports](https://builds.hex.pm/builds/otp/amd64/ubuntu-22.04/builds.txt) OTP 24+